pkg/kubelet/apis/podresources/server.go

```diff
@@ -29,6 +29,12 @@ type DevicesProvider interface {
 	UpdateAllocatedDevices()
 }
 
+// DevicesProvider knows how to provide the devices used by the given container
+type CPUsProvider interface {
+	GetCPUs(podUID, containerName string) []uint32
+	UpdateAllocatedDevices()
```

Comment:
We should name this more appropriately, maybe UpdateAllocatedCPUs(). Also, where is this method implemented? I assumed that it would be implemented in cpu_manager.go but didn't find its implementation

Reply:
Thanks, nice catch, I forget to remove it from here. In DeviceManager this interface is necessary to actualize podDevices state (GetDevice of DeviceManager return devices from podDevices), for CPUManager, where we return cpu directly - UpdateAllocatedDevices interface is not necessary as well as something like podDevices abstraction, since CPUManager uses CheckpointState for such purpose.

Reply:
That sounds reasonable!